data_filtering.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from scipy.signal import savgol_filter
+
+
+def filter_data_coordinates(data):
```

Comment:
I don't know how to understand what this function acually expects in `data`... Is it possible to define it somehow for python? (and is it needed? Hard for me to judge)